registry/session.go

```diff
@@ -331,18 +339,19 @@ func (r *Session) GetRemoteTag(registries []string, repository string, askedTag
 			continue
 		}
 
-		var tagId string
-		if err := json.NewDecoder(res.Body).Decode(&tagId); err != nil {
+		var tagID string
+		if err := json.NewDecoder(res.Body).Decode(&tagID); err != nil {
 			return "", err
 		}
-		return tagId, nil
+		return tagID, nil
 	}
 	return "", fmt.Errorf("Could not reach any registry endpoint")
 }
 
+// GetRemoteTags retrieves all tags from the given repository
```

Comment:
Excellent question, the `result` variable doesn't appear to line up with what is documented https://docs.docker.com/reference/api/hub_registry_spec/#tags-registry